### PR TITLE
Update Netflow readme 

### PR DESCRIPTION
**Pull request by cherryleaf**
Type of change

- Enhancement

## What does this PR do?

Tidy up of Markdown in Readme.md, including adding link to developer's documentation, if missing 

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/netflow/changelog.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # newer versions go on top
 - version: "1.4.1"
   changes:
-    - description: Add documentation for multi-fields
+    - description: Tidy up of Markdown in Readme.md, including adding link to developer's documentation, if missing
```
Comment (bmorelli25) [inline packages/netflow/changelog.yml:4]:
```suggestion
    - description: Added link to Netflow documentation
```

File: packages/1password/_dev/build/docs/README.md [hunk 2]
```diff
@@ -34,4 +34,4 @@ Uses the 1Password Events API to retrieve information about items in shared vaul
 
 {{fields "item_usages"}}
 
-{{event "item_usages"}}
+{{event "item_usages"}}
```
Comment (efd6) [inline packages/1password/_dev/build/docs/README.md:37]: Please revert this removal of a trailing newline.

File: packages/netflow/changelog.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 # newer versions go on top
 - version: "1.4.1"
   changes:
-    - description: Add documentation for multi-fields
+    - description: Added link to Netflow documentation
       type: enhancement
-      link: https://github.com/elastic/integrations/pull/2916
+      link: https://github.com/elastic/integrations/pull/3002
```
Comment (efd6) [inline packages/netflow/changelog.yml:6]: This should be a new section, not an overwrite of the existing last entry.

**Review by efd6 — CHANGES_REQUESTED** [review]

File: packages/1password/_dev/build/docs/README.md (hunk 2 above)
Comment (GinnyGalicia) [reply packages/1password/_dev/build/docs/README.md]: Done

File: packages/netflow/changelog.yml (hunk 3 above)
Comment (GinnyGalicia) [reply packages/netflow/changelog.yml]: Oops. Fixed

File: packages/1password/_dev/build/docs/README.md (hunk 2 above)
Comment (efd6) [reply packages/1password/_dev/build/docs/README.md]: Thank you.

File: packages/netflow/_dev/build/docs/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Netflow Integration
 
-This integration is for receiving NetFlow and IPFIX flow records over UDP. It
+This integration is for receiving [NetFlow](https://docs.netflowlogic.com/) and IPFIX flow records over UDP. It
```
Comment (andrewkroh) [inline packages/netflow/_dev/build/docs/README.md:3]: This is not the related link. Netflow and IPFIX are standards.

- https://www.ietf.org/rfc/rfc3954.txt

- https://www.ietf.org/rfc/rfc7011.txt

**Review by andrewkroh — CHANGES_REQUESTED** [review]

File: packages/netflow/_dev/build/docs/README.md (hunk 4 above)
Comment (GinnyGalicia) [reply packages/netflow/_dev/build/docs/README.md]: I've updated the links

**Comment by andrewkroh** [comment]
> README.md is outdated. Rebuild the package with 'elastic-package build'
> [2022-08-10T04:03:55.675Z] Error: checking package failed: checking readme files are up-to-date failed: checking readme files are up-to-date failed

@GinnyGalicia can you please regenerate the README file with `elastic-package build`.

**Comment by GinnyGalicia** [comment]
Ran build again

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**